types/gavant-ember-validations/index.d.ts

```diff
@@ -2,19 +2,25 @@ import MutableArray from '@ember/array/mutable';
 import Copyable from '@ember/object/-private/copyable';
 import Route from '@ember/routing/route';
 import Controller from '@ember/controller';
+import { BufferedChangeset } from 'validated-changeset/dist/types';
 
 declare global {
     interface Array<T> extends MutableArray<T>, Copyable {}
+
     // Use Subclass<T> to allow abstract and non-abstract subclasses of T
     // Use ConcreteSubclass<T> to allow only non-abstract subclasses of T
     type Subclass<T> = Function & { prototype: T }; // tslint:disable-line ban-types
     type ConcreteSubclass<T> = new(...args: any[]) => T;
+
+    interface ChangesetValidationsMap {
```

Comment:
I feel like this could be expanded upon? Is it really just a string with any property? or does it have to be an object

Reply:
yeah thats no longer needed, as ember-changeset now exports its own `ValidatorMap` type for that.